packages/experiments/src/components/TilesList/TilesList.tsx

```diff
@@ -105,13 +108,13 @@ export class TilesList<TItem> extends React.Component<ITilesListProps<TItem>, IT
   public render(): JSX.Element {
     const { cells } = this.state;
 
-    const { className, onActiveElementChanged, items, cellsPerPage, ref, role, focusZoneComponentRef, ...divProps } = this.props;
+    const { className, onActiveElementChanged, items, cellsPerPage, role, focusZoneComponentRef, ...divProps } = this.props;
 
     return (
       <FocusZone
         role={role}
         {...divProps}
-        ref={ref as ((element: FocusZone | null) => void)}
+        ref={focusZoneComponentRef as ((element: FocusZone | null) => void)}
```

Comment:
This will break our API contract unless I'm misunderstanding. The above is replacing a `HTMLElement` ref with a `componentRef` prop set on the following line:

https://github.com/OfficeDev/office-ui-fabric-react/pull/9439/files#diff-533bdd4a04491852153288ac46951998R118